# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/printing/print-microtask-after-navigate.html.ini
# Commit: edf3df2adebb
# Full Hash: edf3df2adebb5971d458c72078763f3a4f46b9b6
# Author: Jonathan Watt <jwatt@jwatt.org>
# Date: 2019-12-21 21:42:36
# Regressor Bug: 1602410
# File Overlap Count: 3
# Description:
#   Bug 1602410. Make `window.print()` invoke printing via the parent process using a JSWindowActor. r=Gijs,kmag
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D56414
# ==============================================================================

diff -r 238918bf6bea -r edf3df2adebb testing/web-platform/meta/printing/print-microtask-after-navigate.html.ini
--- a/testing/web-platform/meta/printing/print-microtask-after-navigate.html.ini	Sat Dec 21 12:27:40 2019 +0000
+++ b/testing/web-platform/meta/printing/print-microtask-after-navigate.html.ini	Sat Dec 21 12:33:11 2019 +0000
@@ -1,4 +1,5 @@
 [print-microtask-after-navigate.html]
+  disabled: https://bugzilla.mozilla.org/show_bug.cgi?id=1569400#c22
   expected:
     if (os == "android") and not debug: ["ERROR", "OK"]
     if (os == "android") and debug: ERROR